tests/zenko_tests/README.md

```diff
@@ -0,0 +1,121 @@
+# How to run zenko end2end test locally with zenko-operator
+
+_For Backbeat tests please see [Using.md](./node_tests/backbeat/Using.md)_
+## Prerequisite
+
+_All with their latest version_
+- Docker
+- Kubectl
+- Helm
+- Tilt
+- Kind
+- m4
+- git
+
+
+## Deploy zenko environment
+
+### Login to docker registry
+    Zenko-operator deployment will need to access private images from the scality registry.
+
+You will need to connect to Harbor, then go into your user profile to generate a “CLI secret”, which you will user as password
+
+Then login from CLI, using your registry Username and the CLI secret you just generated as password:
+
+
+```shell
+$ docker login registry.scality.com
+User name: <Username on registry.scality.com>
+Password: <Generated CLI secret>
+```
+
+***
+### Download zenko operator from git repository
+
+```shell
+$ git clone https://github.com/scality/zenko-operator.git
+$ cd zenko-operator
+```
+***
+### Adapt config to create ingress for s3api endpoint
+
+Edit file `/doc/examples/zenko-1.2-dev.yaml` by adding this at the end of the file
+
+```yaml
+initialConfiguration:
+    locations:
+    - name: us-east-1
+      type: location-file-v1
+    s3API:
+      endpoints:
+      - hostname: s3.zenko.local
+        locationName: us-east-1
+```
+
+***
+### Change versions to suit your needs
+
+Edit file `/doc/examples/zenkoversion-1.2-dev.yaml` by changing the versions
+
+_For example:_
+```diff
+vault:
+  image: registry.scality.com/vault/vault
+-      tag: '8.3.6'
++      tag: '8.4.0'
+  shell:
+    image: busybox
+
+```
+***
+
+### Run zenko operator
+
+Run the script:
+
+`./hack/scripts/bootstrap-kind-dev.sh`
+
+<hr style="border:2px solid gray"/>
+
+## Configure env to run end2end tests
+
+#### Get access key
+
+```shell
+$ export ZENKO_ACCESS_KEY=$(kubectl get secret dev-management-vault-admin-creds.v1  -n default -o  jsonpath='{.data.accessKey}' | base64 -d)
+```
+
+#### Get secret key
+
+```shell
+$ export ZENKO_SECRET_KEY=$(kubectl get secret dev-management-vault-admin-creds.v1  -n default -o  jsonpath='{.data.secretKey}' | base64 -d)
+```
+
+#### Edit hosts file
+```shell
+$ sudo vi /etc/hosts
+# add this line:
+127.0.0.1  iam.zenko.local ui.zenko.local s3-local-file.zenko.local keycloak.zenko.local sts.zenko.local management.zenko.local s3.zenko.local
+```
+#### Come to your local vaultclient root folder and test connection
+```shell
+$ ADMIN_ACCESS_KEY_ID=$ZENKO_ACCESS_KEY ADMIN_SECRET_ACCESS_KEY=$ZENKO_SECRET_KEY ./bin/vaultclient list-accounts --host iam.zenko.local --port 80
+```
+
+### Export endpoints in env
+
+```shell
+$ export CLOUDSERVER_ENDPOINT=s3.zenko.local
+$ export VAULT_ENDPOINT=iam.zenko.local
+$ export CLOUDSERVER_HOST=s3.zenko.local
+```
```

Comment:
add `export VAULT_STS_ENDPOINT=http://sts.zenko.local:80`
for all these ENDPOINT envs, we'd better add `http://` prefix and `:80` port, as there is a function here https://github.com/scality/Zenko/blob/development/2.3/tests/zenko_tests/node_tests/VaultClient.js#L61
which will split the VAULT_ENDPOINT into host and port. it will raise error if we don't include `http://` and `:80`